content/tutorials/react-setup.md

```diff
@@ -0,0 +1,102 @@
+---
+title: React App Setup
+description: Tutorial to set up a basic React app that uses squid-js.
+---
+
+This tutorial shows how you can build a basic [React](https://reactjs.org/) app that uses the squid-js JavaScript package to publish a data set, get a data set, and more.
+
+## Requirements
+
+- `nodejs` >= 10 is installed. You can check using `node -v`
+- `npm` >= 5.2 is installed. You can check using `npm -v`
+- Running all the services started by `./start_ocean.sh --no-pleuston --local-pond-node` in [oceanprotocol/docker-images](https://github.com/oceanprotocol/docker-images) running with
+- Using browser with [MetaMask](https://metamask.io/) and some Ether in your account (for the network you're connected to). See [the tutorial about getting Ether and Ocean Tokens for testnets](/tutorials/get-ether-and-ocean-tokens/).
+
+## Tutorial Steps
+
+1. Run `npx create-react-app marketplace` in you terminal. This will create a folder named `marketplace` with a boilerplate React app.
+2. Move to your app directory with `cd marketplace` and run `yarn add @oceanprotocol/squid web3`. This adds the OceanProtocol and Web3 packages to the app.
+3. At this point you can already run `yarn start` which starts the app in your browser at [localhost:3000](http://localhost:3000).
+4. To clear the React spinning icon, open `src/App.js` and modify the source to:
+
+   ```javascript
+   import React, { Component } from 'react'
+   import './App.css'
+   class App extends Component {
+     render() {
+       return (
+         <div className="App">
+           <h1>Marketplace app</h1>
+         </div>
+       )
+     }
+   }
+   export default App
+   ```
+
+5. Below the `import './App.css'` line, let's import the packages we installed, set up web3 and unlock MetaMask accounts (if locked):
+
+   ```javascript
+   import { Ocean } from '@oceanprotocol/squid/dist/node/squid'
```

Comment:
is being so specific now required after the recent changes in squid-js? Otherwise should be just `import { Ocean } from '@oceanprotocol/squid`

Reply:
@jernejpregelj You should probably answer that one.

Reply:
@kremalicious 
Which recent changes? We still need to provide web3 object in config and unlock metamask for website if we want to register assets.

Reply:
I mean only the new build output for node & browser and just referring to the long import path `@oceanprotocol/squid/dist/node/squid` which should simply be `@oceanprotocol/squid`

Reply:
@kremalicious pushed commits with this fixed.